docs/premium-features/custom-types-and-actions.md

```diff
@@ -59,70 +61,134 @@ To delete a Module, follow these steps:
+
+#### Enum Name and Description
+
+1. Click on the "Add DTO" button in the top right of the APIs tab.
+2. In the dropdown, click on "Add Enum".
+3. Provide a name for your Enum, such as "MyNewEnum". This will automatically populate the Name field with the provided name.
+4. Optionally, add a description for your Enum.
+
+#### Adding Members
+
+1. In the "Members" section, click on "Add member" to define the Enum's members.
+2. For each member:
+   - Specify the Name of the member.
+   - Provide the Value for the member.
+3. Repeat step 2 for each additional member you want to add to the Enum.
+
+#### Finalizing Enum Creation
+
+Click the "Create Enum" button to finalize the creation process.
+
+After completing these steps, your new Enum will be created within the selected Module, and you can use it as a type for properties in your DTOs or as input/output types for your Actions.
+
+### Edit DTO or Enum
 
-![Edit, Disable, and Delete DTO](./assets/custom-types-and-actions/edit-disable-delete-dto.png)
+To edit an existing DTO, click on the DTO within its associated Module in the APIs tab. You can modify the DTO's name, description, and properties as needed. Remember to save your changes when you're done.
 
 ## Actions
 
 Actions represent the API endpoints that define the operations available in your application. They specify the HTTP methods, paths, input, and output data structures for each endpoint.
 
-Actions use DTOs to define their input and output data. Each Action can have a request DTO (for input) and a response DTO (for output). Amplication generates built-in Actions based on your application's entities, but you can also create custom Actions to implement specific functionality.
+Actions use DTOs to define their input and output data. Each Action can have a request DTO (for input) and a response DTO (for output). Amplication generates built-in Actions based on your application's entities for CRUD operations, but you can also create custom Actions to implement specific functionality.
+
+:::note
+The generated Amplication service has support for HTTP mode only. Our [HTTPs plugin](https://amplication.com/plugins/transport-https) adds support for HTTPS mode and TLS termination at the app level.
+:::
 
 ### Create Action
 
 To create a new Action, follow these steps:
 
+#### Module Name and Description
+
 1. Click on the "Add Action" button in the top right of the APIs tab.
-2. Provide a name for your new Action.
-3. Click the "Create Action" button to finalize the creation process.
+2. Provide a Display Name for your new Action, such as "My New Module Name". This will automatically populate the Name field with a generated name like "MyNewModuleName".
+3. Optionally, add a description for your Action.
+
+#### Choosing Types
+
+1. In the "Types" section, select the input and output types for your Action:
+   - For the Input Type, choose "DTO" from the dropdown and click "Select..." to pick an existing DTO or create a new one.
```

Comment:
Let's add the the Input and Output type can be also primitive, not only DTO, and it can be chosen from the drop-down list